front/src/common-app/components/footer/footer.component.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import * as classes from './footer.styles';
+
+interface Props {}
+
+export const Footer: React.FC<Props> = props => {
+  const {} = props;
+
+  return (
+    <footer className={classes.footer}>
+      <img
+        className={classes.imgFooter}
+        src="./src/assets/logo-lemon.png"
+        alt="lemoncode"
+      />
+      <ul className={classes.menuFooter}>
+        <li className={classes.itemMenuFooter}>
+          <a className={classes.linkMenuFooter} href="#">
+            Aviso legal
+          </a>
+        </li>
+        <li className={classes.itemMenuFooter}>
+          <a className={classes.linkMenuFooter} href="#">
+            Privacidad
+          </a>
+        </li>
+        <li className={classes.itemMenuFooter}>
+          <a className={classes.linkMenuFooter} href="#">
+            Contactar
+          </a>
+        </li>
```

Comment:
Since all links uses the same structure, you could componentize this, for instance, create just in the sae file a subcomponent:

```tsx

interface Props {
  href : string;
}
FooterLinkComponent<props> = (props : Props) => {
  return {href} = props;

  return (
        <li className={classes.itemMenuFooter}>
          <a className={classes.linkMenuFooter} href="{href}">
            {children}
          </a>
        </li>
  )
}
```

Then the parent component will look quite simpler:

```tsx
<footer className={classes.footer}>
      <img
        className={classes.imgFooter}
        src="./src/assets/logo-lemon.png"
        alt="lemoncode"
      />
      <ul className={classes.menuFooter}>
        <FooterLinkComponent url="#">
           Aviso legal
        </FooterLinkComponent>
        <FooterLinkComponent url="#">
           Privacidad
        </FooterLinkComponent>
        <FooterLinkComponent url="#">
            Contactar
        </FooterLinkComponent>
```

Reply:
resolved in new commit